### DISCUSSION TITLE
Last Update 2024/10

### DISCUSSION THREAD

**Discussion by lucaferrucci** [dominikamann/oekofen-pellematic-compact#68] [Q&A]
good morning dominik, after the last update the integration generates an error
![Screenshot 2024-10-03 alle 14 12 32](https://github.com/user-attachments/assets/f7c8e1e0-d4c0-4a66-873d-08e0b68d82da)

**Comment by dominikamann** [comment]
Hi Luca, 

what is displayed when you click on an error?

I think the new units are correct there was a change some months ago see https://github.com/dominikamann/oekofen-pellematic-compact/commit/69e10c93c593e4b66a97887b7fd2096e5303d80e.

So I assume you can ignore/confirm them. 

Dominik

**Comment by lucaferrucci** [comment]
![Screenshot 2024-10-04 alle 11 26 31](https://github.com/user-attachments/assets/87c7407e-f481-4f8f-b0d8-95438f7644f4)
I attach the image, I ignore them but they appear at every restart of hass

**Reply by dominikamann** [reply]
That's strange. Can you see if there are may older statistics for this sensor?
